drake/solvers/QuadraticProgram.m

```diff
@@ -130,7 +130,7 @@
       if(cnstr.num_cnstr~= 1)
         error('Drake:QuadraticProgram:UnsupportedConstraint','QuadraticProgram cannot accept LinearConstraint with multiple rows in its objective')
       end
-      obj.f(xind) = obj.f(xind)+A';
+      obj.f(xind) = obj.f(xind)+cnstr.A';
```

Comment:
Yes, but we have never called `addCost` with a linear cost before. I add a test case that would cover this line.